deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -1,6 +1,6 @@
 # {{ ansible_managed }}
 from prado.hooks import error
-from prod_api_creds import api_key, api_user
+from config.prod_api_creds import api_key, api_user
```

Comment:
shoot, yes, `configs` is right, `config` is wrong.

Once I figured out that `__init__.py` was the problem, I didn't try a relative import. But maybe you're right!